core/controlplane/config/templates/cloud-config-etcd

```diff
@@ -100,7 +105,6 @@ coreos:
         Type=oneshot
         EnvironmentFile=-/etc/etcd-environment
         EnvironmentFile=-/var/run/coreos/etcdadm-environment
-        ExecStartPre=/usr/bin/systemctl is-active {{.Etcd.SystemdUnitName}}
```

Comment:
Yes, this check should be removed. Otherwise we end up with a restarted-and-then-failed etcd member with corrupted data never restored.
It is caused by that fact that `etcdadm-check.service` never run due to unstarted `etcd-member.service`, which results etcdadm to never notice the failure(=/var/run/coreos/etcdadm/member-failure-beggning-time` is never updated)